.jscsrc

```diff
@@ -0,0 +1,111 @@
+{
+	// Project Specific
+	"excludeFiles": [
+		"node_modules/**",
+		"coverage/**",
+	  	"lib/**"
+	],
+
+	// Added for JSX Support
```

Comment:
If you don't need the .jsx extension, just remove this section since the default is `.js` files